### DISCUSSION TITLE
How to use UIWindow (or whatever the window object is)

### DISCUSSION THREAD

**Discussion by TechStudent10** [MyreMylar/pygame_gui#285] [Q&A] [answered]
Hello! I've seen this module being used in a Python operating system called Snakeware and would like to use it in my project. I've looked through the docs, but can't find anything.

I am looking for the object used in the first (pink) screenshot and it's usage.

Thank you in advance!

**Comment by MyreMylar** [comment, marked as answer]
Hello,

Have you looked at the pygame_gui_examples GitHub repo? There are several
examples that make use of the UIWindow in there.

The UIFileDialog and UIMessageWindow windows in the main pygame_gui project
also make use of it.

There is also some auto-generated documentation in the docs you might have
missed.

I hope that helps a bit, I'm on holiday right now so sorry I can't dig out
links directly.

**Reply by TechStudent10** [reply]
Alright I figured it out. Thank you!
